src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { Status } from '../../types/Status';
+import { Filter } from '../Filter';
+
+interface Props {
+  todos: Todo[];
+  filterBy: Status;
+  onAllclick: () => void;
+  onActiveClick: () => void;
+  onCompletedClick: () => void;
+}
+
+export const Footer: React.FC<Props> = React.memo(({
+  todos,
+  filterBy,
+  onAllclick,
+  onActiveClick,
+  onCompletedClick,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todos.filter(todo => !todo.completed).length} items left`}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
```

Comment:
Remove comment. Try to write obvious code with meaningful variable and function naming. Fix all cases